README.md

```diff
@@ -145,7 +145,7 @@ then you may need to force rustbuild to use an older version. This can be done
 by manually calling the appropriate vcvars file before running the bootstrap.
 
 ```batch
-> CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.bat"
+> CALL "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64|x86
```

Comment:
since this is a readme, maybe there should be a bit more details about the meaning of this, or let the old version for desktop mode?

Reply:
Their readme was outdated and referenced VS2015. It did not build with those instructions on latest master.

Just checked and now it seems they partially updated it https://github.com/rust-lang/rust/commit/7171bd1f69fb1c1a999b8e968fc693da325f3cc4, but not the .bat file path. I did that (and added the `x86` in case you want to build the `i686` target).